packages/config/config/devices/0x018c/pm-b400zw-n.json

```diff
@@ -0,0 +1,73 @@
+{
+	"manufacturer": "Dawon DNS",
+	"manufacturerId": "0x018c",
+	"label": "PM-B400ZW-N",
+	"description": "Power Manager",
+	"devices": [
+		{
+			"productType": "0x0042",
+			"productId": "0x0006",
+			"zwaveAllianceId": 1870
+		},
+		{
+			"productType": "0x0042",
+			"productId": "0x0005"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Group 1",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Standy Power Setting Value",
+			"description": "If 300, standby power is 3.00w and standby power is enabled. default : 0, disabled",
```

Comment:
Unnecessary description. Disabled should be an option.

Reply:
Removed description